save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/entities/User.kt

```diff
@@ -6,10 +6,12 @@ import javax.persistence.Entity
  * @property name
  * @property password *in plain text*
  * @property role role of this user
+ * @property source where the user identity is coming from, e.g. "github"
  */
 @Entity
 class User(
     var name: String?,
     var password: String?,
     var role: String?,
+    var source: String,
```

Comment:
It will be a part of primary key for uniqueness?

Reply:
Now I'm considering only user name to be unique. Do you mean we should allow users with the same name from different systems, like from Github and Google? I think it makes sense

Reply:
Added logic to add user identity source to data passed downstream from gateway, as well as uniqueness constraint